packages/xdl/src/detach/IosWorkspace.js

```diff
@@ -182,7 +182,10 @@ async function _renderPodfileFromTemplateAsync(
     TARGET_NAME: projectName,
   };
   let reactNativeDependencyPath;
-  const detachableUniversalModules = Modules.getDetachableModulesForPlatform('ios');
+  const detachableUniversalModules = Modules.getDetachableModulesForPlatformAndSdkVersion(
+    'ios',
+    sdkVersion
+  );
```

Comment:
is it the only place that we should update? I guess we also need to change sth in `android-tasks.js` and `ios-tasks.js`